### ISSUE TITLE
Request for Pre-trained Model for Siamese KPConv Paper

### ISSUE THREAD

**Issue by pingcong** [IdeGelis/torch-points3d-SiamKPConvVariants#2]
I have been following content related to change detection recently and was particularly excited to come across your publication on "Siamese KPConv: 3D multiple change detection from raw point clouds using deep learning." I have been working on a few change detection projects related to 3D point clouds and have observed that the results presented in your article have achieved state-of-the-art performance.

I am very interested in using the network described in your paper to verify on my own dataset. However, I apologize for not being able to find a pre-trained model. I attempted to train the model on the dataset provided in the paper, but due to insufficient computational resources, the training was interrupted at epoch=26.

Would it be possible for you to provide a pre-trained model at your convenience? I believe it would greatly benefit my research efforts. I am looking forward to your reply.

Thank you for your time and consideration.

Best regards,

**Comment by IdeGelis** [comment]
Hello, 

No problem! I wanted to set it public, but the files are too heavy. Here you have a link to download the four developed models (Siamese KPConv, Encoder Fusion SiamKPConv, OneConvFusion and Triplet KPConv) : 
[https://filesender.renater.fr/?s=download&token=eca9d1d9-5bf2-42d7-8f76-5b1b8b9a9b30](https://filesender.renater.fr/?s=download&token=eca9d1d9-5bf2-42d7-8f76-5b1b8b9a9b30)
 These models are trained on Urb3DCD dataset (V2, low density (0.5 pt/m²)). 
I hope this will help you.

Best regards,
Iris

**Comment by pingcong** [comment]
Thank you very much. May I ask you a question? Compared to 2D, 3D annotation has always been time-consuming and laborious. In fact, from a mesh level perspective, it is more accurate. If I want to do some truth data related to 3D data, are there any recommended shortcuts or tools? I am looking forward to your reply.

Best regards!

**Comment by IdeGelis** [comment]
Hello,

Yes, 3D point cloud annotation is always very time-consuming. I used to use CloudCompare software, which lets you group points together and annotate these points all together.

Iris

**Comment by pingcong** [comment]
Okay, Pcl and cloucompare related to point clouds have also been used. Have you considered annotating data from the mesh level? Will this be faster, or are there any pre annotated models that can be referenced to improve efficiency?
Best regards!

**Comment by pingcong** [comment]
Thank you for your attention to these inquiries.，I am pleased to inform you that we have successfully replicated the results based on a pre-trained model, achieving commendable test accuracy and performance metrics. The test results are as follows: test_acc=95.69, test_loss=0.224, test_loss_reg=0.000, test_macc=90.51, test_miou=78.59, test_miou_ch=75.84.

Regarding your queries:
1、I would like to inquire about the origin of the path `/gpfswork/rech/rin/utf14pr/dev/path`. Kindly guide me on where this path is referenced or introduced within our codebase or configuration settings. 
2、Preprocessing Required for Validation on Custom Datasets？？

**Comment by pingcong** [comment]
Thank you for your attention to these inquiries.，I am pleased to inform you that we have successfully replicated the results based on a pre-trained model, achieving commendable test accuracy and performance metrics. The test results are as follows: test_acc=95.69, test_loss=0.224, test_loss_reg=0.000, test_macc=90.51, test_miou=78.59, test_miou_ch=75.84.

Regarding your queries:
1、I would like to inquire about the origin of the path /gpfswork/rech/rin/utf14pr/dev/path. Kindly guide me on where this path is referenced or introduced within our codebase or configuration settings.
2、Preprocessing Required for Validation on Custom Datasets？？
3、without ground truth data

**Comment by coledea** [comment]
Hi,

could you renew the link for the trained models? It has expired by now.
I would much appreciate it.

Best regards

**Comment by IdeGelis** [comment]
Hello,
Here you have a link to download the trained models : [https://filesender.renater.fr/?s=download&token=8f7a6da0-277d-44be-9259-f48385091c12](https://filesender.renater.fr/?s=download&token=8f7a6da0-277d-44be-9259-f48385091c12)

I hope it's not too late for you,
Sorry for the delay.

Best regards,
Iris
